DNN Platform/Tests/DotNetNuke.Tests.Web/Api/DnnActionFilterProviderTests.cs

```diff
@@ -3,24 +3,45 @@
 // See the LICENSE file in the project root for more information
 
 namespace DotNetNuke.Tests.Web.Api
-{
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
     using System.Collections.ObjectModel;
-    using System.Linq;
+    using System.Linq;
+    using System.Reflection;
     using System.Web.Http;
     using System.Web.Http.Controllers;
-    using System.Web.Http.Filters;
-
+    using System.Web.Http.Filters;
+    using DotNetNuke.Common.Utilities;
     using DotNetNuke.Web.Api;
     using DotNetNuke.Web.Api.Internal;
     using Moq;
     using NUnit.Framework;
 
     [TestFixture]
     public class DnnActionFilterProviderTests
-    {
+    {
+        [SetUp]
+        public void SetUp()
+        {
+            var mockCache = new Mock<ICBO>();
+            mockCache
+                .Setup(x => x.GetCachedObject<IEnumerable<PropertyInfo>>(It.IsAny<CacheItemArgs>(), It.IsAny<CacheItemExpiredCallback>(), It.IsAny<bool>()))
+                .Returns(new PropertyInfo[0]);

```

Comment:
This `SetUp` could probably also use a pass-through logic, rather than hardcoding the return value.  Here's a generic setup for `ICBO.GetCachedObject`
```suggestion
            var mockCache = new Mock<ICBO>();
            mockCache
                .Setup(x => x.GetCachedObject<It.IsAnyType>(It.IsAny<CacheItemArgs>(), It.IsAny<CacheItemExpiredCallback>(), It.IsAny<bool>()))
                .Returns(new InvocationFunc(invocation => ((CacheItemExpiredCallback)invocation.Arguments[1])((CacheItemArgs)invocation.Arguments[0])));
```

Reply:
This should probably be put in a test helper class, since it's useful in many places.